Node interop docker image build fails consistently

Fails consistently on master, failing 100% of test runs.
e.g.
https://source.cloud.google.com/results/invocations/4856fb30-685c-4a7e-b0ca-08b064c85298/log

```
  CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.o
make: *** No rule to make target 'Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/lb_policy/subchannel_list.o', needed by 'Release/obj.target/libgrpc.a'.  Stop.
make: Leaving directory '/var/local/git/grpc-node/packages/grpc-native-core/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
...
2018-05-06 08:37:38,861 FAILED: build_docker_node [ret=1, pid=9291, time=464.1sec]
```

--- a/tools/dockerfile/interoptest/grpc_interop_node/build_interop.sh
+++ b/tools/dockerfile/interoptest/grpc_interop_node/build_interop.sh
@@ -17,22 +17,11 @@
 set -e
 
 mkdir -p /var/local/git
-
 git clone /var/local/jenkins/grpc-node /var/local/git/grpc-node
 # clone gRPC submodules, use data from locally cloned submodules where possible
 (cd /var/local/jenkins/grpc-node/ && git submodule foreach 'cd /var/local/git/grpc-node \
 && git submodule update --init --recursive --reference /var/local/jenkins/grpc-node/${name} \
 ${name}')
-
-# Use the pending c-core changes if possible
-if [ -d "/var/local/jenkins/grpc" ]; then
-  cd /var/local/jenkins/grpc
-  CURRENT_COMMIT="$(git rev-parse --verify HEAD)"
-  cd /var/local/git/grpc-node/packages/grpc-native-core/deps/grpc/
-  git fetch --tags --progress https://github.com/grpc/grpc.git +refs/pull/*:refs/remotes/origin/pr/*
-  git checkout $CURRENT_COMMIT
-  git submodule update --init --recursive --reference /var/local/jenkins/grpc
-fi
 
 # copy service account keys if available
 cp -r /var/local/jenkins/service_account $HOME || true

